scripts/build/TestFx.Loc.targets

```diff
@@ -29,23 +29,23 @@
     <CreateItem Include="@(ResxResources)" AdditionalMetadata="Language=%(ResxLang.Identity)">
       <Output ItemName="LocResourceFile" TaskParameter="Include"/>
     </CreateItem>
-    <Exec Command="$(TestFxPackagesRoot)fmdev.xlftool\0.1.3\tools\xlftool.exe update -Resx %(LocResourceFile.Identity) -Xlf $(ResourceDirectory)\xlf\%(LocResourceFile.Filename).%(LocResourceFile.Language).xlf" />
-    <Exec Command="$(TestFxPackagesRoot)fmdev.xlftool\0.1.3\tools\xlftool.exe update -Resx %(ResxResources.Identity) -Xlf $(ResourceDirectory)\xlf\%(ResxResources.Filename).xlf" />
+    <Exec Command="$(TestFxPackagesRoot)fmdev.xlftool.0.1.3\tools\xlftool.exe update -Resx %(LocResourceFile.Identity) -Xlf $(ResourceDirectory)\xlf\%(LocResourceFile.Filename).%(LocResourceFile.Language).xlf" />
```

Comment:
I see both `fmdev.xlftool` and `fmdev.xlftool.0.1.3` paths in packages directory. Could this be a nuget change (may be newer version)?

```powershell
56 $ ls D:\dd\gh\Microsoft\testfx\packages\fmdev*
    Directory: D:\dd\gh\Microsoft\testfx\packages


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----        6/12/2017   5:01 PM                fmdev.xlftool
d-----        6/12/2017   5:01 PM                fmdev.xlftool.0.1.3
```

Reply:
On my PC only single folder generated

```
PS > ls packages\fmdev*

    Каталог: D:\d\user\GitHub\testfx\packages

Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----       10/30/2017   1:36 PM                fmdev.xlftool.0.1.3
```